service-spec/schema-validator/src/main/resources/hprof/0.6.5/profile-spec.json

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
Remove this file from the review.
We can update the .gitignore file in the parent directory to ensure spec files are not included by git.